src/common/selectors/clusterSelectors.ts

```diff
@@ -126,4 +126,6 @@ export const selectSchedulableMasters = (cluster: Cluster): boolean => {
 };
 
 export const isClusterPlatformTypeVM = ({ platform }: Pick<Cluster, 'platform'>) =>
-  !/baremetal|none/.test(platform?.type ?? 'none');
+  SupportedPlatformIntegrations.includes(platform?.type ?? 'none');
+
+export const SupportedPlatformIntegrations: PlatformType[] = ['vsphere', 'nutanix'];
```

Comment:
Doesn't look like this list belongs to the cluster domain, please try to find a more appropriate place for it